### PR TITLE
Firmware update

### PR DESCRIPTION
**Pull request by coKaliszewski**
Firmware update patched vulnerabilities and require MessageId, TimeStamp, and Sign values to be different each time.  A unique key is used when creating the values that comes from Meross when logging in.   You can get the key by using this script 
 https://github.com/bapirex/meross-api/blob/master/login.py

I haven't been able to test this on previous firmware versions since I already updated and don't have another device with an older version.

### CODE REVIEW

File: drivers/meross-smart-wifi-garage-door-opener.groovy [hunk 1]
```diff
@@ -51,50 +53,58 @@ def getDriverVersion() {
...
-        log 'missing setting configuration'
+        log.warn('missing setting configuration')
         return
     }
-
     sendEvent(name: 'door', value: open ? 'opening' : 'closing', isStateChange: true)
 
     try {
-        log 'do sendCommand'
+        def payloadData = currentVersion >= 323 ? getSign() : [MessageId: settings.messageId, Sign: settings.sign, CurrentTime: settings.timestamp]
```
Comment (ithinkdancan) [inline drivers/meross-smart-wifi-garage-door-opener.groovy:78]: what is getSign calling?

Comment (coKaliszewski) [reply drivers/meross-smart-wifi-garage-door-opener.groovy:78]: I committed the wrong project.  The sign method is there and it also checks for null values when getting the version number in the most recent commit.

**Comment by ithinkdancan** [comment]
@coKaliszewski sorry for the delay will try and test this out this week

**Comment by ithinkdancan** [comment]
@coKaliszewski can you rebase and see if https://github.com/ithinkdancan/hubitat-meross/pull/5 covered the firmware update?

**Pull request closed without merging**